pronouncing/__init__.py

```diff
@@ -225,6 +225,7 @@ def rhymes(word):
                                     element), []) if w != word])
         combined_rhymes = list(chain.from_iterable(combined_rhymes))
         combined_rhymes.sort()
-        return combined_rhymes
+                unique_combined_rhymes = set(combined_rhymes)
```

Comment:
There's an unexpected indent here. Should it be inline with the others?

Reply:
Absolutely. Sorry about that. Should be taken care of now.